modules/cf-core/test/global-teardown.jest.ts

```diff
@@ -1,3 +1 @@
-export default async function globalTeardown() {
-  global["chain"].server.close();
-}
+export default async function globalTeardown(): Promise<void> {}
```

Comment:
will these testing infra changes cause problems in upstream CF PRs?

Reply:
They're small changes to just a couple files so merging these changes should be painless.